bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClientImpl.java

```diff
@@ -546,7 +553,7 @@ public void getBookieInfo(final BookieSocketAddress addr, final long requested,
             } else {
                 pcbc.getBookieInfo(requested, cb, ctx);
             }
-        }, requested);
+        }, requested, conf.getUseV2WireProtocol());
```

Comment:
I don't think the value here should be `conf.getUseV2WireProtocol()`. It should be `true /* forceUseV3 *`, right?